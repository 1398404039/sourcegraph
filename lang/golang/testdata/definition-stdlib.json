[
  {
    "id": 0,
    "jsonrpc": "2.0",
    "method": "initialize",
    "params": {
      "rootPath": "testdata",
      "capabilities": {}
    }
  },
  {
    "id": 1,
    "jsonrpc": "2.0",
    "method": "textDocument/definition",
    "params": {
      "textDocument": {
        "uri": "gopath/src/mux/mux.go"
      },
      "position": {
        "line": 39,
        "character": 24
      }
    }
  },
  {
    "id": 2,
    "jsonrpc": "2.0",
    "method": "shutdown",
    "params": null
  }
]
