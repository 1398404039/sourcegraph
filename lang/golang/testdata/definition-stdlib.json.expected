[
  {
    "result": {
      "capabilities": {
        "hoverProvider": true,
        "definitionProvider": true,
        "referencesProvider": true
      }
    },
    "id": "0",
    "jsonrpc": ""
  },
  {
    "result": [
      {
        "uri": "stdlib://da6b9ec7bf1722fa00196e1eadc10a29156b6b28//src/net/http/server.go",
        "range": {
          "start": {
            "line": 56,
            "character": 5
          },
          "end": {
            "line": 56,
            "character": 11
          }
        }
      }
    ],
    "id": "1",
    "jsonrpc": ""
  },
  {
    "result": true,
    "id": "2",
    "jsonrpc": ""
  }
]